packages/common/src/store/pages/chat/selectors.ts

```diff
@@ -128,18 +128,18 @@ export const getHasUnreadMessages = (state: CommonState) => {
   // This really shouldn't be necessary since the above should be kept in sync
   const chats = getChats(state)
   for (const chat of chats) {
+    if (chat.is_blast) {
```

Comment:
thinking out loud - maybe better pattern is to make a new selector `getNonBlastChats` or something. but this works for now.